website/data/community.json

```diff
@@ -342,5 +342,15 @@
       "type": "application-provider",
       "badges": [],
       "description": "<p>NexJ’s Integrated Advisor Desktop acts as a hub connecting best-of-breed solutions to deliver a superior seamless advisor experience that improves productivity, accelerates client acquisition, increases client retention and maximizes relationships. It contains all information about each client and household and integrates to other systems, allowing advisors to complete processes without logging into multiple applications and switching contexts.</p>"
+    },
+    {
+    "title": "FactSet",
+        "publisher": "FactSet",
+        "image": "/img/users/FactSet.png",
```

Comment:
Do you need an image for this? I can provide one if needed.

Reply:
it already exists in the project